storage/boltz/query.go

```diff
@@ -48,11 +49,58 @@ type EntitySymbol interface {
 	Eval(tx *bbolt.Tx, rowId []byte) (FieldType, []byte)
 }
 
```

Comment:
Simplify Map function API by using a context instead of a complicated return signature